qTox does not build on Debian 8 Jessie because of QVector<T>::removeOne() usage

##### Brief Description

OS: Debian 8 KDE
qTox version: HEAD
toxcore:  HEAD
Qt:  5.3.2, stock Debian 8
Hardware:  VirtualBox

Reproducible: Always
##### Steps to reproduce

Build qTox using stock Qt on Debian 8
##### Observed Behavior

```
src/widget/passwordedit.cpp: In member function ‘void PasswordEdit::unregisterHandler()’:
src/widget/passwordedit.cpp:47:31: error: ‘class QVector<QAction*>’ has no member named ‘removeOne’
         eventHandler->actions.removeOne(action);
                               ^
```

As removeOne() was added in [Qt 5.4](https://doc.qt.io/qt-5/qvector.html#removeOne)
##### Expected Behavior

Should build on all supported systems, mentioned on INSTALL.md.

I believe it should be stressed in contribution guidelines to use lowest-common-denominator Qt version. This is not the first time I have reported such incompatibility error.

--- a/src/widget/passwordedit.cpp
+++ b/src/widget/passwordedit.cpp
@@ -44,7 +44,8 @@ void PasswordEdit::unregisterHandler()
 #ifdef ENABLE_CAPSLOCK_INDICATOR
     if (eventHandler && eventHandler->actions.contains(action))
     {
-        eventHandler->actions.removeOne(action);
+        //TODO: future: use removeOne() when Qt 5.3 (Debian 8) support ends.
+        eventHandler->actions.remove(eventHandler->actions.indexOf(action));
         if (eventHandler->actions.isEmpty())
         {
             delete eventHandler;

